relay-general/src/store/transactions.rs

```diff
@@ -153,9 +152,7 @@ impl Processor for TransactionsProcessor {
             ));
         }
 
-        if span.op.value().is_none() {
-            return Err(ProcessingAction::InvalidTransaction("span is missing op"));
-        }
+        span.op.get_or_insert_with(|| "default".to_owned());
```

Comment:
Discussed with the team on what the default value should be and we settled on `"default"`.

We avoided "undefined", "null" and similar terms because they could be confused with an UI error.
We considered how it will appear on searches, `transaction.op:default` seemed better than `transaction.op:op`.

@dashed @untitaker @HazAT